src/components/ui/input.tsx

```
@@ -0,0 +1,22 @@
+import * as React from "react"
+
```

ESLint

[error] Missing semicolon.

(semi)